features/step_definitions/form_step_steps.rb

```diff
@@ -20,3 +20,7 @@
     assert_text "Add #{::Regexp.last_match(1).humanize(capitalize: false)}"
   end
 end
+
+Then("I should see a button labelled {string}") do |label|
+  expect(page).to have_button(label)
+end
```

Comment:
There's a Rubocop linting failure here - the easiest thing do do is run `bundle exec rubocop -A` to autofix any issues. We should probably have this in a pre-commit hook

Reply:
D'oh, just seen you'd fixed it :)